src/js/features/compile.js

```diff
@@ -21,23 +22,126 @@
     const dynamicTextarea = document.querySelector('.CodeMirror textarea');
     dynamicTextarea.value = staticTextarea.value;
     button.textContent = '컴파일';
-    button.addEventListener('click', (evt) => {
+    await button.addEventListener('click', (evt) => {
```

Comment:
```suggestion
    button.addEventListener('click', async (evt) => {
```

이렇게만 수정하시면 됩니다.

`await button...` 은 의도하신 대로 동작하지 않을거에요. 이벤트를 등록하는 것을 await 하는 것이지, 등록된 함수를 await 하지 않습니다.